.gitignore

```diff
@@ -0,0 +1,24 @@
+# Built application files
+*.apk
+*.aar
+*.aab
+
+# Delvik VM files
+*.dex
```

Comment:
does dex show up somewhere else?

Reply:
Shows up under `build/intermediates` so I thought I'd separate it out  from the `Built application files` section since it's not set of build files, not a final built application file